Source/Core/Core/IOS/DI/DI.cpp

```diff
@@ -48,7 +48,7 @@ IPCCommandResult DI::IOCtl(const IOSIOCtlRequest& request)
     StartIOCtl(request);
 
   // DVDInterface handles the timing and we handle the reply,
-  // so WII_IPC_HLE shouldn't handle anything.
+  // so IOS shouldn't handle anything.
```

Comment:
What I meant by this comment is that WII_IPC_HLE.cpp (at least I think it was that file...) wouldn't handle anything, but the wording here is making it sound like IOS as a whole (including this DI code) will do nothing at all.